_data/customers.yml

```diff
@@ -1,71 +1,71 @@
 - name: Bind
   logo: /assets/img/logos-clients/logo-bind.png
 
-- name: Gridpoint
-  logo: /assets/img/logos-clients/logo-gridpoint.png
+- name: Creative Assembly
+  logo: /assets/img/logos-clients/logo-creative-assembly.png
```

Comment:
Please remove the logos assets for the old clients.

Reply:
I removed all from the attached list. 
<img width="143" alt="Zrzut ekranu 2023-07-5 o 09 11 04" src="https://github.com/gruntwork-io/gruntwork-io.github.io/assets/72336345/5a26a685-b4e7-42ab-a966-4b6cf8533a86">